src/main/java/tyrone/command/CommandHandler.java

```diff
@@ -0,0 +1,157 @@
+package tyrone.command;
+
+import tyrone.savemanager.FileReadWriter;
+import tyrone.command.exceptions.EmptyFieldException;
+import tyrone.task.Deadline;
+import tyrone.task.Event;
+import tyrone.task.TaskList;
+import tyrone.task.Todo;
+
+public class CommandHandler {
+
+    public static boolean isExitCommand (String command) {
+        return command.equals("bye");
+    }
+
+    public static void handleCommand(String command) {
+        String[] dissectedCommand = command.split(" ");
+        if (command.equals("list")) {
+            handleList();
+        } else if (dissectedCommand[0].equals("mark")) {
```

Comment:
Maybe you could refactor `dissectedCommand[0]` to indicate it the command, and the rest are arguments?